surveys/2021-annual-survey/questions.md

```diff
@@ -713,6 +713,7 @@ Select one:
 - Doesn't add a feature I want
 - Stagnation
 - Instability of the language
+- Development of the language not scalable/sustainable
```

Comment:
```suggestion
- Development of the language is not scalable
```

I think having an answer here specifically about scaling is good. I think sustainability basically covers all of the answers, so this answer is better off without it.